ajsonrpc/scripts/server.py

```diff
@@ -49,6 +49,12 @@ def handle_task_result(self, task):
         logger.info('Close the client socket')
         self.transport.close()
 
+# Compatible with cyfunction.
+# This change helps to work with some modules like `pydantic`, which wraps
+# functions (includes coroutine functions) into cyfunctions. The method
+# `inspect.isfunction()` cannot recognize them for now.
+def is_func_or_cyfunc(object):
```

Comment:
@FrostyLeaf could you please add the description underneath the function as pydoc suggests?